cmd/ecs-discovery/main.go

```diff
@@ -25,6 +25,7 @@ import (
 	"time"
 
 	log "github.com/sirupsen/logrus"
+	"golang.org/x/term"
```

Comment:
Is this required due to the Golang update to version 1.17?

Reply:
Nope, but we did this for one the repos, so for consistency, decided to do it everywhere else too. You will find there is a lot of commonality in the codebase for this stack :)

Reply:
Sounds very sensible.